packages/twenty-front/src/hooks/useTableSort.ts

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+
+enum sortOrders {
```

Comment:
We already have `OrderBy`

Reply:
Update the type to OrderBy. thanks for pointing it out.